### DISCUSSION TITLE
v3.11.0-rc.0

### DISCUSSION THREAD

**Discussion by janzandr** [EnMAP-Box/enmap-box#130] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/EnMAP-Box/enmap-box/releases/tag/v3.11.0-rc.0'>v3.11.0-rc.0</a>.</em>

**Comment by janzandr** [comment]
Hi @jakimowb and @thielfab, please check out RC0 Zip.

**Comment by janzandr** [comment]
Dear @aloboa, FYI - we will upload Releases and Pre-Releases from now on here on GitHub.
Feel free to test the v3.11.0-rc.0 version, which we plan to release very soon.

**Comment by aloboa** [comment]
Great, thanks.
By "here" do you mean https://github.com/EnMAP-Box/enmap-box/releases ?
Not easy to find...

**Reply by janzandr** [reply]
Yes, sorry. Users have to get used to the structure, I guess.
